library/rocksdbjni/RocksDbBuilder.kt

```diff
@@ -2,28 +2,28 @@ package library.rocksdbjni
 
 import org.zeroturnaround.exec.ProcessExecutor
 import org.zeroturnaround.exec.ProcessResult
-import java.io.File
+import java.nio.file.Path
 import java.nio.file.Paths
 
 fun main() {
     val baseDir = Paths.get(".")
     val version = Paths.get("library").resolve("rocksdbjni").resolve("VERSION").toFile().useLines { it.firstOrNull() }
-    val javaHome = "/Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home"
+    val javaHome = Paths.get(shellScript("/usr/libexec/java_home", baseDir, null)!!.outputUTF8().trim())
```

Comment:
What does this line do? Specifically the `!!.`

Reply:
That's how you invoke a method on a variable that may be `null`. I also didn't like it because it looks terse.

Let me try to find if there's a clearer way to do it (maybe by preceding the call with some kind of `assertNotNull` or something like that)

Reply:
Actually we can clean it!! The original function returns a `Nullable` but actually it doesn't need to. Fixed in https://github.com/graknlabs/dependencies/pull/213/commits/7f55e0882571e8475a475660447200569483cba6